core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -446,7 +446,7 @@ public void testConcurrentFastAppends(@TempDir File dir) throws Exception {
                 final int currentFilesCount = numCommittedFiles;
                 Awaitility.await()
                     .pollInterval(Duration.ofMillis(10))
-                    .atMost(Duration.ofSeconds(10))
+                    .atMost(Duration.ofSeconds(20))
```

Comment:
I'm not sure increasing the timeout is the right thing to do here. Isn't this an issue due to exponential retries during concurrent commits? I'd probably rather define/decrease https://github.com/apache/iceberg/blob/ea5da1789f3eb80fc3196bcdd787a95ac0c493ba/core/src/main/java/org/apache/iceberg/TableProperties.java#L89-L92 for this test

Reply:
Good point. I'd also increase the number of retries.